### ISSUE TITLE
viewport meta tag doesn't reflect current best use for iOS

### ISSUE THREAD

**Issue by e110c0** [joshbuchea/HEAD#198]
with the iPhone X being over a year old and the iPad now having safe areas as well, changing the default viewport tag to something like `'initial-scale=1, viewport-fit=cover'` would be more appropriate.

**Comment by scottaohara** [comment]
thanks!

**Comment by joshbuchea** [comment]
Hey gentlemen, I'm not sure that `viewport-fit=cover` is a good default or that removing `shrink-to-fit=no` is necessary. Do either of you have any references to support these changes being better defaults? I'm going to go ahead and revert for now. In the future I would prefer to discuss changes to the recommended defaults. Thanks!

**Comment by e110c0** [comment]
I did not expect that change to just be merged TBH. I'd rather start a discussion and find out what a good default is.

Adding it to the documentation of what can be done and what it is for might be the better solution.

**Comment by scottaohara** [comment]
yup.  i jumped the gun here by not providing reasoning why i merged the change.

Upon review / testing on iOS devices running v10.3, v11.x and 12.1 `shrink-to-fit` is no longer recognized or needed.

I'll be posting a write up about this soon.

**Comment by scottaohara** [comment]
Report: https://www.scottohara.me/blog/2018/12/11/shrink-to-fit.html
tldr; `shrink-to-fit=no` became irrelevant during the iOS 9.3 lifecyle and beyond

**Comment by scottaohara** [comment]
FWIW: [pull request to remove shrink-to-fit from HTML5 boiler plate](https://github.com/h5bp/html5-boilerplate/pull/2103)

**Comment by joshbuchea** [comment]
Apologies for the late response, open source has been low on my list of priorities lately. Excellent writeup @scottaohara 👌 I'm on-board, let's remove `shink-to-fit=no`.

**Comment by joshbuchea** [comment]
Also — FWIW @e110c0 I think we definitely should include `viewport-fit=cover` somewhere, just not sure if it belongs in the recommended minimum as it seems there are reports of it breaking websites by covering navigation and other controls.

**Comment by scottaohara** [comment]
no worries @joshbuchea :)

**Comment by joshbuchea** [comment]
@scottaohara I'm not sure how large this dataset is but, iOS **9.0-9.2.x** usage is only @ **0.17%** https://data.apteligent.com/ios/

**Comment by scottaohara** [comment]
oh that's interesting.  thank you @joshbuchea.  going to update my article with that link

**Comment by joshbuchea** [comment]
Oh yeah and almost forgot — @scottaohara thank you for doing that research and for publishing your findings 💯

**Comment by scottaohara** [comment]
i like to share :)

**Comment by e110c0** [comment]
@scottaohara totally agree it's not a recommended minimum.

Having all possible options in the documentation would be nice. I still haven't figured out all pitfalls of it, so not comfortable with creating a PR at the moment
